### PR TITLE
ProductBundle Controller + Views

### PR DESCRIPTION
**Pull request by jdewit**
Created basic controller and views

### CODE REVIEW

**Comment by iampersistent** [comment]
I'm going to wait until I get the models for the product up so this will have something to work with, that will be the next thing I do after the MonetaryBundle implementation

File: app/logs/dev.log [hunk 1]
```diff
@@ -0,0 +1,258 @@
+2011-03-01T14:34:23-08:00 DEBUG: Notified event "core.request" to listener "Symfony\Bundle\FrameworkBundle\RequestListener::handle" (notifyUntil)
+2011-03-01T14:34:23-08:00 INFO: Matched route "homepage" (parameters: {"_controller":"Symfony\\Bundle\\FrameworkBundle\\Controller\\RedirectController::urlRedirectAction","url":"\/vespolina\/web\/app_dev.php\/","permanent":true,"_route":"homepage"})
```
Comment (stof) [inline app/logs/dev.log:2]: Log files should not be in the repo

**Comment by iampersistent** [comment]
@jdewit do you have time to rebase this against the ProductBundle repo, clean up the log file (above), and do a pull request for that bundle?

**Comment by stof** [comment]
and btw, files should probably stay with 644 permissions instead of making them executables.

**Comment by iampersistent** [comment]
I merged this into iampersistent/ProductBundle and I'll merge that into vespolina/ProductBundle later today

**Comment by iampersistent** [comment]
I merged this into IamPersistent/ProductBundle and I'll merge that into vespolina/ProductBundle later today

**Pull request closed without merging**